week-1-practise/level-1/5-logic-error-2.js

```diff
@@ -1,7 +1,7 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return word.trim;
+  return word.trim();
```

Comment:
Correct! Did you take note of what `word.trim` returned?

Reply:
yes. it removed unnecessary spaces